src/bin/inx-chronicle/cli.rs

```diff
@@ -231,12 +232,29 @@ impl ClArgs {
                         // Panic: Acceptable risk
                         res.unwrap()?;
                     }
-                    return Ok(true);
+                    return Ok(PostCommand::Exit);
+                }
+                #[cfg(debug_assertions)]
+                Subcommands::ClearDatabase { no_run } => {
+                    println!("Connecting to database at bind address `{}`.", config.mongodb.conn_str);
+                    let db = chronicle::db::MongoDb::connect(&config.mongodb).await?;
+                    db.clear().await?;
+                    println!("Database cleared successfully.");
+                    if *no_run {
+                        return Ok(PostCommand::Exit);
+                    }
+                }
+                Subcommands::BuildIndexes => {
+                    println!("Connecting to database at bind address `{}`.", config.mongodb.conn_str);
+                    let db = chronicle::db::MongoDb::connect(&config.mongodb).await?;
+                    super::build_indexes(&db).await?;
+                    println!("Indexes built successfully.");
```

Comment:
It's kind of weird to have a mixture between `println!` and `tracing` statements (`info!` within `super::build_indexes` for example). I agree that CLI commands lend themselves more to `println!` statements, but I don't see any way to support this in all called functions without introducing a lot of overhead. Should we switch to `tracing::info!` here instead then?

Reply:
Yeah, probably.